examples/routing_service_shapes_processor/cpp/CMakeLists.txt

```diff
@@ -33,16 +44,27 @@ find_package(
 add_library(shapesprocessor
     "${CMAKE_CURRENT_SOURCE_DIR}/ShapesProcessor.cxx")
 
+# Set target properties for lang requirement output library name
+set_target_properties(shapesprocessor
+    PROPERTIES
+        CXX_STANDARD 11
+        OUTPUT_NAME_DEBUG shapesprocessord
+        LIBRARY_OUTPUT_DIRECTORY "${output_dir}"
```

Comment:
Do you need to specify all the configurations? I mean, if you set ``LIBRARY_OUTPUT_DIRECTORY`` you shouldn't need to set ``LIBRARY_OUTPUT_DIRECTORY_RELEASE`` . If ``LIBRARY_OUTPUT_DIRECTORY_RELEASE`` is not set, CMake will try to use ``LIBRARY_OUTPUT_DIRECTORY``.

Reply:
We discussed this online and we agreed we need them